sechub-commons-model/src/test/java/com/mercedesbenz/sechub/commons/model/SecHubConfigurationModelValidatorTest.java

```diff
@@ -21,10 +23,106 @@ class SecHubConfigurationModelValidatorTest {
     private static final String VALID_NAME_WITH_MAX_LENGTH = "---------1---------2---------3---------4---------5---------6---------7---------8";
     private static final String VALID_NAME_BUT_ONE_CHAR_TOO_LONG = VALID_NAME_WITH_MAX_LENGTH + "-";
     private SecHubConfigurationModelValidator validatorToTest;
+    private SecHubConfigurationModelSupport modelSupport;
+    private Set<ScanType> modelSupportCollectedScanTypes;
 
     @BeforeEach
     private void beforeEach() {
+
+        modelSupportCollectedScanTypes = new LinkedHashSet<>();
+        modelSupportCollectedScanTypes.add(ScanType.CODE_SCAN); // just one
```

Comment:
When I remove this, 31 of 42 tests do fail... because no scan types are found for the module group...

But I will remove it from the `beforeEach` method and put it dedicated to the tests, so it is more clear that it is necessary.